### PR TITLE
added a Baudrate Variable to config for Baudrate selection

### PR DESCRIPTION
**Pull request by Haerteleric**
Hi i added a config var to specify a baudrate, cause the upcoming conbee III will run at 115200 baud/s

### CODE REVIEW

File: zigpy_deconz/config.py [hunk 1]
```diff
@@ -26,13 +26,20 @@
 CONF_WATCHDOG_TTL = "watchdog_ttl"
 CONF_WATCHDOG_TTL_DEFAULT = 600
 
+CONF_DEVICE_BAUDRATE = "baudrate"
+
+SCHEMA_DEVICE = SCHEMA_DEVICE.extend(
+    {vol.Optional(CONF_DEVICE_BAUDRATE, default=115200): int}
```
Comment (puddly) [inline zigpy_deconz/config.py]: The default baudrate I think should remain 38400, as changing it globally will break existing installs.

Instead, we should update `ControllerApplication.probe` to try both baudrates and return the correct one.

**Review by puddly — CHANGES_REQUESTED** [review]
Thanks!

Is the serial protocol backwards compatible?

File: zigpy_deconz/config.py (hunk 1 above)
Comment (Haerteleric) [reply zigpy_deconz/config.py]: I agree with the default baudrate change, it should be 38400  (Raspbee I/II and ConBee I run on 38400, the ConBee II doesn't care about the used baudrate, as it uses a USB-CDC driver). Could you provide me with a reference for `ControllerApplication.probe`, so i can research this ?

BTW: Check your Discord DMs we (phoscon) want to send you a ConBee III free of charge

**Comment by Haerteleric** [comment]
i changed the default Baudrate to 38400 for backwards compatibility

**Review by puddly — APPROVED** [review]

**Pull request merged**